gobblin-data-management/src/main/java/gobblin/data/management/copy/writer/FileAwareInputStreamDataWriter.java

```diff
@@ -230,6 +231,8 @@ public boolean apply(FileStatus input) {
         } else {
           log.info("File {} copied.", copyableFile.getOrigin().getPath());
         }
+        this.state.setProp(SlaEventKeys.SOURCE, copyableFile.getOrigin().getPath().toString());
```

Comment:
Shouldn't the source be the source cluster as opposed to the fully qualified path?